buildstockbatch/gcp/gcp.py

```diff
@@ -16,13 +16,17 @@
 :license: BSD-3
 """
 import argparse
+from datetime import datetime
 import docker
 import json
 import logging
 import os
 import pathlib
 import re
 
+from google.cloud import batch_v1
+
+from buildstockbatch.aws.aws import DockerBatchBase
```

Comment:
Why is this importing AWS' DockerBatchBase (and how does that work when there's already a DockerBatchBase in this file)?